src/components/core.ts

```diff
@@ -190,16 +190,23 @@ export default class Core {
     /**
      * Check if holderId is not empty
      */
-    if (!this.config.holderId) {
-      throw Error('«holderId» param must being not empty');
+    if (!this.config.holderId && this.config.holder) {
```

Comment:
I guess here is a typo

```suggestion
    if (!this.config.holderId && !this.config.holder) {
```

Reply:
thx